src/components/music/ChordExercise.js

```diff
@@ -134,6 +137,13 @@ class ChordExercise extends React.Component {
             ? "green"
             : "red"
           : "",
+        icon: this.state.answerWasWrong ? (
```

Comment:
Could this somehow be combined into an object (eg. `{color: "", icon: (<>)} `) with `borderColor` above to avoid duplicate tenary

Same thing in all Exercise files